cmd/nerdctl/compose_run_linux_test.go

```diff
@@ -432,12 +432,13 @@ func TestComposePushAndPullWithCosignVerify(t *testing.T) {
 	keyPair := newCosignKeyPair(t, "cosign-key-pair")
 	defer keyPair.cleanup()
 
-	reg := testregistry.NewPlainHTTP(base, 5000)
-	defer reg.Cleanup()
+	reg := testregistry.NewWithNoAuth(base, 0, false)
+	defer reg.Cleanup(nil)
+
 	localhostIP := "127.0.0.1"
 	t.Logf("localhost IP=%q", localhostIP)
 	testImageRefPrefix := fmt.Sprintf("%s:%d/",
-		localhostIP, reg.ListenPort)
+		localhostIP, reg.Port)
```

Comment:
Why renamed?

Reply:
"ListenIP" makes sense (0.0.0.0) - as in "the address the registry process is binding to" - and also does "IP" (the exposed ip you can access it with).
"ListenPort" feels the wrong way - since this is NOT the port the registry process is actually listening on (still 5000), but instead the port you can access it with.

It seems better to me that way.

Either way, this is minor. I can reverse it if you dont like it.

Thanks @AkihiroSuda